pkg/sign/certificate.go

```diff
@@ -75,6 +81,14 @@ type chain struct {
 }
 
 func NewFulcio(opts *FulcioOptions) *Fulcio {
+	if opts.Client == nil {
+		client := &http.Client{}
+		if opts.Timeout != 0 {
```

Comment:
If no timeout is provided I would still configure a timeout (30s?), as the default client blocks "forever" and that can lead to very unexpected behaviours.